gradle-incremental-configuration-cache/src/main/java/com/palantir/gradle/configcache/incremental/IncrementalConfigurationCachePlugin.java

```diff
@@ -38,19 +50,25 @@ public final void apply(Project project) {
 
         Path allowListPath = project.getRootProject().getProjectDir().toPath().resolve(ALLOW_LIST_FILE);
         if (!Files.exists(allowListPath)) {
-            throw new GradleException(
-                    String.format("Configuration cache allowed tasks file not found at %s", allowListPath));
+            throw new GradleException(String.format(
+                    "Configuration cache allow list not found at %s\n%s", allowListPath, ALLOW_LIST_INFO));
         }
 
         AllowListFile allowList = new AllowListFile(allowListPath);
-        Set<String> enabledTasks = allowList.loadAllowedTasks();
-
-        project.getAllprojects().forEach(proj -> proj.getTasks().configureEach(task -> {
-            if (!enabledTasks.contains(task.getPath())) {
-                task.notCompatibleWithConfigurationCache(String.format(
-                        "Configuration cache is not enabled for this task, as it was not included in %s",
-                        ALLOW_LIST_FILE));
-            }
-        }));
+        try {
+            Set<String> enabledTasks = allowList.loadAllowedTasks();
+            project.getAllprojects().forEach(proj -> proj.getTasks().configureEach(task -> {
+                if (!enabledTasks.contains(task.getPath())) {
+                    task.notCompatibleWithConfigurationCache(String.format(
+                            "Configuration cache is not enabled for this task, as it was not included in %s",
+                            ALLOW_LIST_FILE));
+                }
+            }));
+        } catch (IOException e) {
+            throw new GradleException(
+                    String.format("Error reading the allow list at %s\n%s", allowListPath, ALLOW_LIST_INFO));
+        }
```

Comment:
Is something happening here with the diff? I feel this PR includes stuff not relevant to the change (it would just be the log basically?)